clippy_lints/src/formatting.rs

```diff
@@ -22,6 +22,28 @@ declare_clippy_lint! {
     "suspicious formatting of `*=`, `-=` or `!=`"
 }
 
+declare_clippy_lint! {
+    /// **What it does:** Checks the formatting of a unary operator on the right hand side
```

Comment:
NIT:
```suggestion
    /// **What it does:** Checks the formatting of an unary operator on the right hand side
```

Reply:
I believe "a unary" is correct in this case due to how it's pronounced.
https://forum.wordreference.com/threads/a-an-university-student-utopia-unilateral-power-u.1661586/

Reply:
Ah TIL, thanks! :)